### PR TITLE
Homepage

### LINKED ISSUE #41: Start building the home screen
I sign in

I should be directed to a home screen, not the courses screen
The home screen should include a recap of the last scorecard that I *completed* (meaning I played all 18 holes of)
The home screen should include a link to any current scorecards I'm playing (this isn't fully fleshed out on the backend).

A current scorecard includes: any scorecard that I haven't played all of, or any scorecard I haven't "marked as complete" (this is a boolean on rounds, I believe)

screen shot 2014-11-05 at 1 47 11 pm

### PR DESCRIPTION
**Pull request by canes6288**
[Closes #41]

### CODE REVIEW

**Comment by gschorkopf** [comment]
@killpack @pdougall1 feel free to peruse this one as well!

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.1.3
+2.1.2
```
Comment (killpack) [inline .ruby-version]: What's this change for?

**Comment by killpack** [comment]
Build seems to be failing due to the Ruby version change that was made in this PR.

**Comment by gschorkopf** [comment]
I believe that was a mistake commit.

File: .ruby-version (hunk 1 above)
Comment (gschorkopf) [inline .ruby-version]: @canes6288 mind reverting this change? Though we're now stable at 2.2.0; perhaps we can update our ruby and ember builds in a future commit, if we're being bold.

**Comment by gschorkopf** [comment]
Closing for now.

**Pull request closed without merging**